ui/partials/main/jobs.html

```diff
@@ -16,8 +16,10 @@
     <span class="result-set-left">
       <span class="result-set-title-left">
         <span>
-          <a href="{{::revisionResultsetFilterUrl}}"
+          <a ng-href="{{::revisionResultsetFilterUrl}}"
+             ng-click="changeResultSet(repoName, resultset.revision)"
              title="View only this resultset"
+             prevent-default-on-left-click
```

Comment:
Hrm, actually, perhaps you COULD leave the ``revisionResultsetFilterUrl`` as one-way binding, but the ``filterParams`` as two-way, since that's the only part that would be changing in the digest cycle.  Anyway, does that make sense?